src/utils/storage/index.ts

```diff
@@ -1,12 +1,9 @@
-import { ImmortalStorage, IndexedDbStore, LocalStorageStore } from 'immortal-db'
+import { ImmortalStorage, LocalStorageStore } from 'immortal-db'
 
 import { getNetworkName } from 'src/config'
 import { Errors, logError } from 'src/logic/exceptions/CodedException'
 
-// Don't use sessionStorage and cookieStorage
-// https://github.com/gruns/ImmortalDB/issues/22
-// https://github.com/gruns/ImmortalDB/issues/6
-const stores = [IndexedDbStore, LocalStorageStore]
+const stores = [LocalStorageStore]
```

Comment:
@gnosis/safe-web FYI, I'm disabling IndexedDB here, we'll use only localStorage.

Reply:
I'd announce this on Slack if I were you.

Reply:
Why? The purpose of using immortaldb was to store data more reliable by spreading them across different storages. If we're going to use only localstorage, we don't need immortaldb at all anymore

Reply:
We'll nuke it completely at some point, but for now disabling IndexedDB is an easy way to make testing/debugging much easier.